spec/recipes/mistral_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+
+describe 'stackstorm::mistral' do
+  before do
+    global_stubs_include_recipe
+  end
+
+  context 'with default node attributes' do
+    let(:chef_run) { ChefSpec::SoloRunner.new.converge(described_recipe) }
+
+    it 'should include recipe openstack-mistral' do
+      allow_any_instance_of(Chef::Recipe).to receive(:include_recipe).with('openstack-mistral')
+      chef_run
+    end
+
+    it 'should create mysql_service "default"' do
```

Comment:
Another informational note (*not directly related to this PR really*) is that [according to installation docs](https://docs.stackstorm.com/install/deb.html) `PostgreSQL` is used by `st2mistral` package as a default DB.

We switched from `MySQL` for a long time ago, meaning it's not officially supported from StackStorm's side.